### ISSUE TITLE
Add caption to image on Exhibition Page Template

### ISSUE THREAD

**Issue by rosy1280** [emory-libraries/Pattern-Library#373]
The exhibition team suggested a caption be added underneath the featured image that appears in the right hand column of an [exhibition template](http://template.library.emory.edu/styleguide/patternlibrary/current/?p=templates-exhibition).

**Comment by kristianserrano** [comment]
@rosy1280 I did a walk through with Clare and Will for the caption on the Exhibition page.
We...
1. identified the figure component we needed to edit,
2. looked at what the figure templating was doing,
3. identified the data structure at the component level and at the template level, and
4. set up the template to pass the appropriate content through to the component.

While I have the changes saved locally, they chose to try making the changes themselves to see if they can do it on their own. I offered to assist if they needed some guidance.

@CB987 and @wulman16 just putting a reminder that we need to include both the caption _and_ the credit if that data is provided.

This change is mainly on the Pattern Library itself; there's no change to the CSS or JS. To get it to display on the website, the Velocity in Cascade needs to have similar conditions for that content. The conditions for the insertion of the `<figcaption>` element will need to be applied at [line 78 of this template in Cascade](https://cascade.emory.edu/entity/open.act?id=d1ebf1fd0ae720982bbcdefa6bc6d5bf&type=format).
